Tests/doc_generator.py

```diff
@@ -0,0 +1,203 @@
+        doc += '| --- | --- | --- |\n'
+        if len(data_map['script']['commands'][i]['arguments']) != 0:
+            for j in range(len(data_map['script']['commands'][i]['arguments'])):
+                doc += '| ' + data_map['script']['commands'][i]['arguments'][j]['name'] + ' | '
+                if not (data_map['script']['commands'][i]['arguments'][j].get('description', False)):
+                    return_error("Error! You are missing description in input " + data_map['script']['commands'][i]['arguments'][j]['name'] +\
+                    " of command " + data_map['script']['commands'][i]['name'])
+                    sys.exit(0)
+                doc += data_map['script']['commands'][i]['arguments'][j]['description'] + ' | '
+                doc += str(data_map['script']['commands'][i]['arguments'][j].get('required', 'False')) + ' | \n'
+        else:
+            doc += '-\n'
+        # Context output
+        doc += '##### Context Output\n'
+        if 'outputs' in data_map['script']['commands'][i]:
+            doc += '| **Path** | **Type** | **Description** |\n'
+            doc += '| --- | --- | --- |\n'
+            for k in range(len(data_map['script']['commands'][i]['outputs'])):
+                doc += '| ' + data_map['script']['commands'][i]['outputs'][k]['contextPath'] + ' | '
+                doc += str(data_map['script']['commands'][i]['outputs'][k].get('type', 'unknown')) + ' | '
+                if not (data_map['script']['commands'][i]['outputs'][k].get('description', False)):
+                    return_error("Error! You are missing description in output " + data_map['script']['commands'][i]['outputs'][j]['name'] + \
+                    " of command " + data_map['script']['commands'][i]['name'])
+                    sys.exit(0)
+                doc += data_map['script']['commands'][i]['outputs'][k]['description'] + ' | \n'
+        else:
+            doc += 'There is no context output for this command.\n'
+
+        # Raw output:
+        doc += '##### Command Example\n'
+        doc += '##### Context Example\n'
+        doc += '##### Human Readable Output\n'
+        doc += '\n'
+    except Exception as e:
+        return_error("Error encountered in the processing of command {} error was missing a {}. Please check your command inputs and outputs".format(data_map['script']['commands'][i]['name'], str(e)))
+        sys.exit(0)
+# Additional info
+doc += '\n## Additional information:\n'
+
+# Known limitations
+doc += '\n## Known limitations:'
+
+# Troubleshooting
+doc += '\n## Troubleshooting:\n'
+
+# Possible Errors
+# if 'True' == demisto.args()['withErrors']:
+#     doc += '\n## Possible Errors (DO NOT PUBLISH ON ZENDESK):\n'
+#     doc += add_error_lines(data_map['script']['script'], data_map['script']['type'])
+
+filename = os.path.basename(sys.argv[1]).replace('documentation-', 'integration-').replace('.yml', '.txt').replace('.yaml', '.txt')  # strip all the spaces
```

Comment:
Lets save the documentation file in other location, perhaps outside the repo so that developers won't accidentally commit these changes to the repo.

Reply:
where is that option, couldn't find it